modules/quotas-sample-resource-quotas-def.adoc

```diff
@@ -117,15 +115,13 @@ spec:
     pods: "2" <1>
     limits.cpu: "1" <2>
     limits.memory: "1Gi" <3>
-    limits.ephemeral-storage: "1Gi" <4>
   scopes:
-  - Terminating <5>
+  - Terminating <4>
 ----
 <1> The total number of pods in a terminating state.
 <2> Across all pods in a terminating state, the sum of CPU limits cannot exceed this value.
 <3> Across all pods in a terminating state, the sum of memory limits cannot exceed this value.
-<4> Across all pods in a terminating state, the sum of ephemeral storage limits cannot exceed this value.
-<5> Restricts the quota to only matching pods where `spec.activeDeadlineSeconds >=0`. For example, this quota would charge for build or deployer pods, but not long running pods like a web server or database.
+<4> Restricts the quota to only matching pods where `spec.activeDeadlineSeconds >=0`. For example, this quota would charge for build or deployer pods, but not long running pods like a web server or database.
```

Comment:
- First sentence: Should there be a space before 0?
- Second sentence: Replace "would charge" with "charges."

Reply:
The first bullet point is left as is, it already exists and I am not sure If i should change that.
Second one >> implemented